### PR TITLE
Close #26 edit question

### LINKED ISSUE #26: Task Edit Question

### PR DESCRIPTION
**Pull request by HaiTranVar**

### CODE REVIEW

File: app/views/user_mailer/account_activation.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<h1 class="text-center"><%= t ".title" %></h1>
```
Comment (HaiTranVar) [inline app/views/user_mailer/account_activation.html.erb]: delete this file

File: app/views/users/_form.html.erb [hunk 2]
```diff
@@ -1,26 +1,17 @@
-<%= render "shared/error_messages", object: user %>
-<%= form_for user do |f| %>
-  <div class="form-group">
-    <%= f.label :name, t(".name") %>
-    <%= f.text_field :name, class: "form-control", value: user.name %>
-  </div>
+<div class="row">
```
Comment (HaiTranVar) [inline app/views/users/_form.html.erb]: why you change this file?

File: app/controllers/questions_controller.rb [hunk 3]
```diff
@@ -27,6 +27,7 @@ def get_question
   end
 
   def question_params
-    params.require(:question).permit :content, answers_attributes: [:id, :content]
+    params.require(:question).permit :content,
```
Comment (dinhngochien) [inline app/controllers/questions_controller.rb]: you need wrap parameter in `()` => `permit(...)`

File: app/controllers/account_activations_controller.rb [hunk 4]
```diff
@@ -1,17 +1,22 @@
 class AccountActivationsController < ApplicationController
-
+  before_action :active_account, only: %i[edit]
```
Comment (dinhngochien) [inline app/controllers/account_activations_controller.rb]: remove redundant file from commit

**Pull request merged**